### ISSUE TITLE
Ask GitHub support to break the fork relationship.

### ISSUE THREAD

**Issue by jtnord** [chrisant996/clink#109]
As this is now the canonical repo for clink it would be helpful if the fork relationship to `mridgers/clink` was broken.

Forked repos are not searchable - and so functionality is lost due to the fork being there.

Please consider contacting GH support to ask them to break the fork linking

**Comment by chrisant996** [comment]
It's something to consider.  I'm not ready to pull the trigger, and I want to understand all the pros and cons.  I suspect there are likely some non-obvious or subtle drawbacks, and I want the opportunity to make a reasoned choice.

For example, I would want to ask github if there's a way to get an exception and keep the fork relationship but also gain searching.  Since this is purely a web front end issue, it is relatively low on my radar right now, but I'll leave the issue open for consideration.

**Comment by chrisant996** [comment]
I've contacted github support and tried to contact Martin.  It seems he's been missing from the internet for 5 years, and he kept a low profile in the first place.  I've gotten only error notifications from my attempts to contact him to get the fork relationship reversed (which was github's recommendation for the best way to proceed).  Without his explicit approval, the only option is to sever the fork relationship.

So I've requested for github support to sever the fork relationship.  😢
